testng-core/src/main/java/org/testng/reporters/jq/Main.java

```diff
@@ -103,11 +103,7 @@ public void generateReport(
 
   private InputStream load(String fileName) {
```

Comment:
Why did we have to change this logic?

Reply:
because that way it works here and it's one specialty less

Reply:
I agree. It is easier.